.gitignore

```diff
@@ -31,6 +31,7 @@ docker-compose.yml
 init-defaultworker
 init-dockerworker
 init-hydroshare
+.vscode/init-hydroshare-debug
```

Comment:
Wondering if we should ignore all of `.vscode`.  I'd rather not have IDE-specific configuration checked in.  I think it largely depends on how much this simplifies setting up debugging in vscode.  If it is simple enough to provide configuration instructions within vscode, that might be preferable.

Reply:
Seems reasonable to me. I'd be happy to write up instructions and keep the .vscode dir that I added here in a different repo in case someone wanted to clone them